recipes/esl.rb

```diff
@@ -21,6 +21,8 @@
 
 case node['platform_family']
 when 'debian'
+  package 'apt-transport-https'
```

Comment:
This was necessary to get tests passing and didn't seem to be part of the base images. I figured pulling it in would be a no-op for most things but still catch when it's not as well.